### ISSUE TITLE
Question about Early Overfitting in GeDEX-TTS Training with VCTK

### ISSUE THREAD

**Issue by thunn** [winddori2002/DEX-TTS#5]
Hello,

I recently ran the GeDEX-TTS training using the VCTK dataset and followed the instructions provided in the GeDEX-TTS/config/VCTK/base.yaml configuration file. The only modifications I made were:

- Changed the `vocoder` to`bigvgan`
- Updated the `vocoder path` to `bigvgan/bigvgan_22khz_80band`

While plotting the training logs, I observed that the model started to overfit very early in the training process. This is somewhat unexpected to me, and I wanted to inquire whether this behaviour is expected or if there might be an issue with my setup.

Here is a basic plot of the training losses.

![GeDex-TTS on VCTK](https://github.com/user-attachments/assets/fef42f80-b891-46af-a556-15cb9a477134)

Could you please provide insights into whether early overfitting is a known issue with this configuration, or if there are any recommended adjustments to prevent this?

Thank you for your help!

**Comment by winddori2002** [comment]
Hi,

Have you listened to the synthesized samples?

The duration loss can increase at the validation phase.

Although the diffusion loss does not improve a lot in training phase, more steps usually guarantee better sample quality as in Grad-TTS.

Thanks.
